### ISSUE TITLE
Receiving an error when trying to view Confluence Chat Add-on Histories.

### ISSUE THREAD

**Issue by leroyllamas** [muchino/confluence.chat#106]
When using the Confluence Chat 2.0.4. When I try and view Chat Histories, I get the standard Atlassian error page: 

![confluence chat view chat histories error](https://cloud.githubusercontent.com/assets/10983191/6177182/9b84853c-b2c1-11e4-810e-2c8b35335455.PNG)
![confluence chat view chat histories error 2](https://cloud.githubusercontent.com/assets/10983191/6177515/b1e19b64-b2c4-11e4-8cc0-3d2f2225b78c.PNG)

**Comment by muchino** [comment]
Thank you for reporting this error. Can you please send me complete log
files?

Thank you
Oli

leroyllamas <notifications@github.com> schrieb am Do., 12.02.2015, 22:37:

**Comment by leroyllamas** [comment]
Thanks Oli. Is there a way to Attach a file to this issue? Or I can send the zip support file to your email address. Let me know what is best and provide your email address if that is how you want me to get you the file. Thanks.

**Comment by muchino** [comment]
Just send the zip to o.straesser.atlassian@gmail.com

leroyllamas <notifications@github.com> schrieb am Fr., 13.02.2015, 16:45:

**Comment by leroyllamas** [comment]
Zip file sent via email to Oli. Thanks.

**Comment by leroyllamas** [comment]
Can I get a status on this error research? Thanks.

**Comment by muchino** [comment]
I apologize, but at the moment I can't find time during the week, so I hope that I can do it this weekend.

Sorry for such a delay

**Comment by leroyllamas** [comment]
Thanks for the update. I found a workaround for now to see my own Chat History. I look forward to hearing from you next week. Thanks again.

**Comment by muchino** [comment]
I suppose there is a SQL error, but sadly I didn't receive your logs - I searched in spam but didn't find something. 
maybe it's enough if you post your complete stacktrace within this thread

**Comment by leroyllamas** [comment]
I emailed the file again since I cannot attach a zip file here.

**Comment by muchino** [comment]
I'm really sorry, bit I didn't receive any mail on o.straesser.atlassian@gmail.com from you, maybe you can share it via dropbox or gdrive?

**Comment by leroyllamas** [comment]
Hi Oli, did you receive it this time?

Leroy Llamas, MBA
Business Analyst II
New Mexico Mutual
P 505.343.2812 | F 505.348.9812 | LeroyL@NewMexicoMutual.com <mailto:scottl@newmexicomutual.com>
www.NewMexicoMutual.com<www.newmexicomutual.com>

**Comment by muchino** [comment]
no

2015-02-23 18:33 GMT+01:00 leroyllamas <notifications@github.com>:

**Comment by leroyllamas** [comment]
Ok. Do you have a dropbox area to place the file?

Leroy Llamas, MBA
Business Analyst II
New Mexico Mutual
P 505.343.2812 | F 505.348.9812 | LeroyL@NewMexicoMutual.com <mailto:scottl@newmexicomutual.com>
www.NewMexicoMutual.com<www.newmexicomutual.com>

**Comment by muchino** [comment]
I hoped you have a https://www.dropbox.com/ account :) because I can't give
you access to my dropbox, so that you can upload that file.

I'm only interested in the whole stack strace, because I hope that there is
something like: Invalid SQL Statement...

2015-02-23 18:36 GMT+01:00 leroyllamas <notifications@github.com>:

**Comment by leroyllamas** [comment]
Where in these folders would I find the Stack Trace:

[cid:image001.png@01D04F56.376E1F50]
Let me know where I can find this for you.

Thanks.

Leroy Llamas, MBA
Business Analyst II
New Mexico Mutual
P 505.343.2812 | F 505.348.9812 | LeroyL@NewMexicoMutual.com <mailto:scottl@newmexicomutual.com>
www.NewMexicoMutual.com<www.newmexicomutual.com>

**Comment by muchino** [comment]
I need the atlassian-confluence.log file (can't see your image)

2015-02-23 18:48 GMT+01:00 leroyllamas <notifications@github.com>:

**Comment by leroyllamas** [comment]
Attached the Confluence log file via email. Oli verified receipt of the log file this AM.

**Comment by muchino** [comment]
Hi, I updated the version in the marketplace, where I hopefully log a bit more to find the solution. 

If all is as wished, you will recieve a nice error message after seraching for histories amd in the log file will be a more detailed stack trace. 

I need this new strack trace (something like "Error while SQL/Hibernate execution")

**Comment by leroyllamas** [comment]
Thanks Oli. My Confluence server is having trouble reaching the Atlassian Marketplace, so I downloaded the .jar file from
a separate browser window.

Do I move this file to the Confluence server and run it from there?

Thanks.

Leroy Llamas, MBA
Business Analyst II
New Mexico Mutual
P 505.343.2812 | F 505.348.9812 | LeroyL@NewMexicoMutual.com <mailto:scottl@newmexicomutual.com>
www.NewMexicoMutual.com<www.newmexicomutual.com>

**Comment by muchino** [comment]
just install it manually by uploading the file in the plugin manager of
confluence

2015-02-25 21:43 GMT+01:00 leroyllamas <notifications@github.com>:

**Comment by leroyllamas** [comment]
Since my server cannot reach the Atlassian Marketplace server, I can't use the plugin manager in Confluence to download the file.

Leroy Llamas, MBA
Business Analyst II
New Mexico Mutual
P 505.343.2812 | F 505.348.9812 | LeroyL@NewMexicoMutual.com <mailto:scottl@newmexicomutual.com>
www.NewMexicoMutual.com<www.newmexicomutual.com>

**Comment by muchino** [comment]
But you can download the file here:
https://marketplace.atlassian.com/download/plugins/confluence.chat/version/20150225
and then upload it in the plugin manager, this is possible without
marketplace access

2015-02-25 21:46 GMT+01:00 leroyllamas <notifications@github.com>:

**Comment by leroyllamas** [comment]
Thanks, I got it uploaded and will run my testing shortly and get you the log file.

Thanks again for the guidance.

Leroy Llamas, MBA
Business Analyst II
New Mexico Mutual
P 505.343.2812 | F 505.348.9812 | LeroyL@NewMexicoMutual.com <mailto:scottl@newmexicomutual.com>
www.NewMexicoMutual.com<www.newmexicomutual.com>

**Comment by leroyllamas** [comment]
Here is the new error. I didn't get the standard Atlassian Error page, just this:
[cid:image001.png@01D0510E.42AA3F70]
Let me know if you need any additional information.

Thanks.

Leroy Llamas, MBA
Business Analyst II
New Mexico Mutual
P 505.343.2812 | F 505.348.9812 | LeroyL@NewMexicoMutual.com <mailto:scottl@newmexicomutual.com>
www.NewMexicoMutual.com<www.newmexicomutual.com>

**Comment by leroyllamas** [comment]
New error after installing the latest Chat version:
![image](https://cloud.githubusercontent.com/assets/10983191/6381860/a78af010-bd01-11e4-9877-0c3e5ee5d1e8.png)

**Comment by muchino** [comment]
Are you using mssql?  And maybe the Confluence schema isn't the default for the Confluence user?

**Comment by leroyllamas** [comment]
Yes we are using MSSQL. Where would I check to see if the Confluence Schema is the default for the Confluence user?

Thanks.

Leroy Llamas, MBA
Business Analyst II
New Mexico Mutual
P 505.343.2812 | F 505.348.9812 | LeroyL@NewMexicoMutual.com <mailto:scottl@newmexicomutual.com>
www.NewMexicoMutual.com<www.newmexicomutual.com>

**Comment by muchino** [comment]
Hard to explain, you haven't a database Admin?  After you logged in into
you mssql database with the client, you may see the scheme in front of the
table name and the default scheme is defined in the settings of the user

leroyllamas <notifications@github.com> schrieb am Mi., 25.02.2015, 23:26:

**Comment by leroyllamas** [comment]
Yes, I will ask my SQL DBA to check this for me since I do not have access to the DB.

Thanks again.

Leroy Llamas, MBA
Business Analyst II
New Mexico Mutual
P 505.343.2812 | F 505.348.9812 | LeroyL@NewMexicoMutual.com <mailto:scottl@newmexicomutual.com>
www.NewMexicoMutual.com<www.newmexicomutual.com>

**Comment by muchino** [comment]
I think it won't be the default for the used user.

But Anyway I have to check if I can recognize this in my add-on..  But I
don't know if it is possible.

leroyllamas <notifications@github.com> schrieb am Do., 26.02.2015, 20:11:

**Comment by leroyllamas** [comment]
The user we have for the Confluence schema is called "confluenceuser" Does that make a difference? Thanks.

**Comment by leroyllamas** [comment]
Hello Oli. Can I get a status on this issue? Do we need to add a user called "confluence" with a default
database scheme in MSSQL of "confluence" ? Thanks.

**Comment by muchino** [comment]
yes this would help to fix your issue

**Comment by EinsteinGit** [comment]
I get the same error with MySQL: 
Error while ShaQL execution: Invalid object name .confdb.bandana'.

I was not getting this problem when I was using the built-in HSQLDB database.
